opencog/relex/Dockerfile

```diff
@@ -0,0 +1,98 @@
+#
+# Docker file that builds RelEx and starts the RelEx server.
+#
```

Comment:
Ok, since the repo is mainly for developers that works. :-) And i agree the
simpler the configuration the better.


On Thu, Feb 26, 2015 at 11:59 AM Linas Vepštas <notifications@github.com>
wrote:

> In opencog/relex/Dockerfile
> <https://github.com/opencog/docker/pull/23#discussion_r25403387>:
>
> > @@ -0,0 +1,98 @@
> > +#
> > +# Docker file that builds RelEx and starts the RelEx server.
> > +#
>
> Or we could keep things manual. The only thing that is scarier to me than
> duplicated code is automatically-generated rpms and debs :-) I mean, all
> hail rolling releases but... I want to minimize complexity. Things are
> already hard-enough to debug.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/docker/pull/23/files#r25403387>.
>